src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -2327,6 +2331,229 @@ protected override void HandleSetBreakpointsRequestAsync(IRequestResponder<SetBr
             responder.SetResponse(response);
         }
 
+        protected override void HandleDataBreakpointInfoRequestAsync(IRequestResponder<DataBreakpointInfoArguments, DataBreakpointInfoResponse> responder)
+        {
+            if (responder.Arguments.Name == null)
+            {
+                responder.SetError(new ProtocolException("SetDataBreakpointRequest failed: Missing 'breakpoints'."));
+                return;
+            }
+
+            DataBreakpointInfoResponse response = new DataBreakpointInfoResponse();
+            string name = responder.Arguments.Name;
+            IDebugStackFrame2 frame = null;
+            IDebugProperty2 property = null;
+            string errorMessage = null;
+
+            // Did our request come with a parent object?
+            if (responder.Arguments.VariablesReference.HasValue)
```

Comment:
>if [](http://example.com/codeflow?start=12&length=2)

I will repeat the suggestion here too -- rather than re-evaluating, I wonder if we want to keep a collection of <parent,name>->evaluation result to avoid needing to reenumerate children to return this info (and other requests that work the same way).